docs/project/copyright.md

```diff
@@ -1,14 +1,14 @@
 Copyright
 =========
 
-The .NET Core copyright is held by ".NET Foundation and Contributors".
+The .NET copyright is held by ".NET Foundation and Contributors".
```

Comment:
Is this correct? (It may be, but I want to double-check.)

Reply:
Great question ...

I would do two things:

- Link to the actual [license](https://github.com/dotnet/runtime/blob/master/LICENSE.TXT) to make the whole thing concrete.
- Use the similar phrasing as is used in the following project ... "The .NET project ..."